Makefile

```diff
@@ -51,6 +51,10 @@ venv: requirements.txt requirements-dev.txt
 install-hooks: venv
 	$(BIN)/pre-commit install -f --install-hooks
 
+.PHONY: lint
+lint:
+	pre-commit run --all-files
```

Comment:
I think you want `$(BIN)/pre-commit` here, since it isn't necessarily installed system-wide but should be installed in the repo's `venv`.